src/auto/useNavigationAuto.ts

```diff
@@ -188,6 +188,15 @@ export const useNavigationAuto = (): {
       moveCamera: (cameraPosition: CameraPosition) => {
         return NavAutoModule.moveCamera(cameraPosition);
       },
+
+      setPadding: (
```

Comment:
Not sure about this order.
**iOS** UIEdgeInsetsMake has this order: _top, left, bottom, right_.
on [**Android** api](https://developers.google.com/android/reference/com/google/android/gms/maps/GoogleMap#setPadding(int,%20int,%20int,%20int)) the order is: _left, top, right, bottom_.

With css (react users could be familiar with this) the order would be: _top | right | bottom | left_

Should we go with css order, or with Android (or keep the current iOS ordering).

@ArturoSalazarB16 @caio1985 ?

Reply:
I update the interface so that values are given with Padding interface, allowing any order.